test/integration/conftest.py

```diff
@@ -127,7 +127,7 @@ def test_case(api_database_with_test_container):
 
 
 @pytest.fixture(scope="module")
-def api_default_database_with_test_conainer_module(
+def api_default_database_with_test_container_module(
```

Comment:
Could we please also fix the names in the fixture?
* `provider` -> `context`
* `db` -> `env` to match the type hint `ExaslctDockerTestEnvironment` which claims to be some sort of an _environment_, rather than a pure `db` (database).
* Maybe we should also use kwargs in the provider/context to make clear what is set to `None` here.